src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -107,41 +107,19 @@ export default function FormNav(props) {
         setIndex(index - 1);
       }
 
-      return () => {
-        // Check main toggle to enable custom focus; the unmounting of the page
-        // before the review & submit page may cause the customScrollAndFocus
-        // function to be called inadvertently
-        if (
-          !(
-            page.chapterKey === 'review' ||
-            window.location.pathname.endsWith('review-and-submit')
-          )
-        ) {
-          if (
-            formConfig.useCustomScrollAndFocus &&
-            (page.scrollAndFocusTarget || formConfig.scrollAndFocusTarget)
-          ) {
-            customScrollAndFocus(
-              page.scrollAndFocusTarget || formConfig.scrollAndFocusTarget,
-              index,
-            );
-          } else {
-            focusByOrder([defaultFocusSelector, 'h2']);
-          }
-        } else {
-          // h2 fallback for confirmation page
-          focusByOrder([defaultFocusSelector, 'h2']);
-        }
-      };
+      // Focus on review & submit header
+      if (!hideFormNavProgress && page.chapterKey === 'review') {
+        scrollTo('topScrollElement');
+        // Focus on review & submit page h2 in stepper
+        focus(
```

Comment:
I think we might still want to keep the h2 fallback just in case. 
maybe - 
- query the dom to see if `va-segmented-progress-bar`
if it exists, check if it has `header-level` and respect that value, then proceed with this code
- otherwise target h2? ( I think even a normal focus needs a little bit of timeout)